lib/db-manager.js

```diff
@@ -445,3 +446,12 @@ var dbManager = module.exports = {
    // @TODO closeStalePulls:
 
 };
+
+/**
+ * Return a copy of the specified array with nulls removed
+ */
+function filterNulls(array) {
+   return array.filter(function(element) {
+      return element;
```

Comment:
Kind of pedantic, but I feel like this should return `element !== null` or the function should be called `filterFalsy`.